pkg/controllers/routing/network_routes_controller.go

```diff
@@ -716,7 +714,11 @@ func (nrc *NetworkRoutingController) startBgpServer() error {
 	go nrc.bgpServer.Serve()
 
 	g := bgpapi.NewGrpcServer(nrc.bgpServer, nrc.nodeIP.String()+":50051"+","+"127.0.0.1:50051")
-	go g.Serve()
+	go func() {
```

Comment:
Not sure if there's a better way to handle errors here.

Reply:
From looking at the code, it looks like `Serve()` only ever returns `nil`.

Since it doesn't matter that we aren't handling this error well in the current version of our GoBGP library and in order to keep this refactor simpler, I would add a TODO message here saying that when we upgrade GoBGP we should re-evaluate the error handling here.

Reply:
If we don't handle this error, the linting won't pass. I'll added a TODO.